### PR TITLE
Part2

### PR DESCRIPTION
**Pull request by razvysme**
First attempt.

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -302,12 +297,12 @@ void MixerChannel::mute(bool muteButton)
...
         outputGain = 1.0f;
 
-    muteButton=!muteButton;   FIXME add spaces to make this easy to read.
+    muteButton =! muteButton;
 }
 void MixerChannel::solo(bool soloButton, int channelNumber)
 {
     number = channelNumber;
-    soloButton=!soloButton; FIXME add spaces to make this easy to read.
+    soloButton =! soloButton;
```
Comment (matkatmusic) [inline main.cpp]: - [ ] the `!` should go next to the variable you're trying to get the opposite value of.

File: main.cpp [hunk 2]
```diff
@@ -349,11 +344,11 @@ void outputChannel::sendToHeadphone( MixerChannel headphones )
 */
 void Mixer::boot()
 {
-    for(int i=0; i<4; ++i) FIXME add spaces to make this easy to read.
+    for( int i=0; i<4; ++i ) 
     {
         monoChannels[i].channel.outputGain = 1.0f; // i know this is crazy but heck, this mixer has no faders
     }
-    for(int i=0; i<2; ++i) FIXME add spaces to make this easy to read.
+    for( int i=0; i<2; ++i )
```
Comment (matkatmusic) [inline main.cpp]: - [ ] add spaces around your `=` and `<>` operators.

File: main.cpp (hunk 1 above)
Comment (razvysme) [reply main.cpp]: Got it.

File: main.cpp (hunk 2 above)
Comment (razvysme) [reply main.cpp]: Yessir

**Comment by razvysme** [comment]
Fixed the issues.

**Pull request merged**